src/sentry/static/sentry/app/views/apiNewToken.jsx

```diff
@@ -109,6 +109,7 @@ const TokenForm = React.createClass({
         <fieldset>
           <MultipleCheckboxField
             key="scopes"
+            name="scopes"
```

Comment:
we could possibly make name optional, as it doesnt seem to REALLY be needed unless we're using form context (and I dont think the browser minds)